docs/reference/aggregations/bucket/geohashgrid-aggregation.asciidoc

```diff
@@ -191,6 +191,62 @@ var bbox = geohash.decode_bbox('u17');
 --------------------------------------------------
 // NOTCONSOLE
 
+==== Requests with additional bounding box filtering
+
+The `geohash_grid` aggregation supports an optional `bounds` parameter
+that restricts the points considered to those that fall within the
+bounds provided. The `bounds` parameter accepts the bounding box in
+all the same <<query-dsl-geo-bounding-box-query-accepted-formats,accepted formats>> of the
+bounds specified in the Geo Bounding Box Query. This bounding box can be used with or
+without an additional `geo_bounding_box` query filtering the points prior to aggregating.
+It is an independent bounding box that can intersect with, be equal to, or be disjoint
+to any additional `geo_bounding_box` queries defined in the context of the aggregation.
+
+[source,console,id=geohashgrid-aggregation-with-bounds]
+--------------------------------------------------
+POST /museums/_search?size=0
+{
+    "aggregations" : {
+        "tiles-in-bounds" : {
+            "geohash_grid" : {
+                "field" : "location",
+                "precision" : 8,
+                "bounds": {
+                  "top_left" : "u17",
```

Comment:
Having the same geohash for `top_left` and `bottom_right` is a bit confusing from an API standpoint. I think if a user explicitly defines `top_left` and `bottom_right` we should expect lat,lon decimal degrees. For example:
```
"bounds" : {  
    "top_left" : "52.71, 4.89", 
    "bottom_right" : "52.69, 4.91" 
} 
```
If the user defines by way of a geohash it should be something simple like `"bounds"` defined as a string instead of an object. For example, `"bounds" : "u17"` and then we compute the spatial bounds of the geohash cell.

Reply:
I just figured I would use this as the example. both are supported. I'll update to use the lat/lon